clientmount.spec

```diff
@@ -0,0 +1,32 @@
+%undefine _missing_build_ids_terminate_build
+%global debug_package %{nil}
+
+Name: dws-clientmount
+Version: 1.0
+Release: 1%{?dist}
+Summary: Client mount daemon for data workflow service
+
+Group: 1
+License: Apache-2.0
+URL: https://github.com/HewlettPackard/dws
```

Comment:
Should we need reference the NearNodeFlash organization instead of HewlettPackard?

Reply:
clientmountd is part of DWS, so it could be used outside of NearNodeFlash. I think that makes HewlettPackard correct unless you disagree.

Reply:
Makes sense, I'm good with that.